src/main/scala/nl/knaw/dans/easy/fedoratobag/DDM.scala

```diff
@@ -62,8 +64,8 @@ object DDM extends DebugEnhancedLogging {
        { /* instructions for reuse not specified as such in EMD */ }
        { emd.getEmdCreator.getDcCreator.asScala.map(bs => <dc:creator>{ bs.getValue.trim }</dc:creator>) }
        { emd.getEmdCreator.getEasCreator.asScala.map(author => <dcx-dai:creatorDetails>{ toXml(author)} </dcx-dai:creatorDetails>) }
-       { dateCreated.map(node =>  <ddm:created>{ node.text }</ddm:created>) }
-       { dateAvailable.map(node =>  <ddm:available>{ node.text }</ddm:available>) }
+       { if (dateCreated.nonEmpty)  <ddm:created>{ dateCreated.toSeq.head }</ddm:created> }
```

Comment:
`.head` is not safe and causes an un-descriptive exception when nothing is available.

Reply:
doesn't the `nonEmpty` take care of that?

Reply:
```suggestion
       { dateCreated.headOption.map(node =>  <ddm:created>{ node.text }</ddm:created>) }
```